src/pretix/base/services/mail.py

```diff
@@ -52,10 +52,11 @@ class SendMailException(Exception):
     pass
 
 
-def mail(email: str, subject: str, template: Union[str, LazyI18nString],
-         context: Dict[str, Any]=None, event: Event=None, locale: str=None,
-         order: Order=None, position: OrderPosition=None, headers: dict=None, sender: str=None,
-         invoices: list=None, attach_tickets=False, auto_email=True, user=None, attach_ical=False):
+def mail(email: Union[str, Sequence[str]], subject: str, template: Union[str, LazyI18nString],
+         context: Dict[str, Any] = None, event: Event = None, locale: str = None,
+         order: Order = None, position: OrderPosition = None, headers: dict = None, sender: str = None,
+         invoices: Sequence = None, attach_tickets=False, auto_email=True, user=None, attach_ical=False,
+         attach_cached_files: Sequence = None):
```

Comment:
Again, I autoformatted here, but pep 8 seems to recommend spaces around the `=` with type annotations.